packages/quantic/force-app/main/default/lwc/quanticNotifications/quanticNotifications.html

```diff
@@ -6,8 +6,8 @@
   <template lwc:else>
     <template for:each={notifications} for:item="notification">
       <template lwc:if={notification.visible}>
-        <div key={notification.id} data-test="notification" class="slds-notify_container slds-is-relative notification-container">
-          <div data-cy="notification" class="slds-notify slds-notify_toast notification" role="status">
+        <div key={notification.id} data-testid="notification" class="slds-notify_container slds-is-relative notification-container">
```

Comment:
Tes pas mieux de prendre notification-container a la place d'un testid ici vue que tu l'a déjà? 🤷

Reply:
The point of `testid` is that if ever the CSS class(es) changes the tests shouldn't necessarily fail.